common/cmake/devices.cmake

```diff
@@ -904,10 +904,12 @@ function(ADD_FPGA_TARGET)
     set(VPR_ARCH_ARGS "")
   endif()
 
+  set(OUT_NOISY_WARNINGS ${OUT_LOCAL}/noisy_warnings.log)
   separate_arguments(
     VPR_BASE_ARGS_LIST UNIX_COMMAND "${VPR_BASE_ARGS}"
     )
   list(APPEND VPR_BASE_ARGS_LIST --route_chan_width ${ROUTE_CHAN_WIDTH})
+  list(APPEND VPR_BASE_ARGS_LIST --suppress_warnings ${OUT_NOISY_WARNINGS},sum_pin_class:check_unbuffered_edges:load_rr_indexed_data_T_values:check_rr_node:trans_per_R)
```

Comment:
Let's do this in the arch define, rather than globally.

Reply:
How can I set the output file in the arch_define.cmake to be in the build directory of a specific test?

Reply:
Two ways I can see.  One is to model VPR_ARCH_ARGS like PLACE_TOOL_CMD (see https://github.com/SymbiFlow/symbiflow-arch-defs/pull/858/files/c70a93d58f07f764e45ff6ad9fbc017fd4eac8ff..ed4d055b1066571749bf9fb6902474c6e829c5db#diff-2d565728ea2fbe9d3a017347718a0ba5R969 ), the other is to fully model warning suppression as a first class argument to define_arch

Reply:
I think the first one could be the most correct way to go as it maintains more consistency with the other commands definitions. I'll proceed with that one then.

Reply:
Done

Reply:
Done